src/components/menu/Menu.js

```diff
@@ -61,6 +61,14 @@ function Menu(props) {
   );
 }
 
+
+/*
+
+  PGM: I think i wrote the below export. It should be simplified to:
+  export default withAsyncAction("auth", "logout")(Menu)
+
+  */
+
```

Comment:
APZ: I honestly have no idea why this was changed from the original code.

Reply:
Yeah pretty sure I messed this up, honestly I shouldnt have even commented on it. It works, its just inconsistent with the way the other exports are written.